about.html

```diff
@@ -0,0 +1,108 @@
+            <div class="contact-info">
+                <p>Please contact us per Email for any further question about BIM Academy!</p>
+                <span class="email">bim-academy@gmail.com</span>
+            </div>
+        </section>
+        <section class="emphasic">
+            <h3>BIM Academy is the training arm of the Institution of Civil Engineers</h3>
+            <hr>
+            <p>Whether you are new or aspiring to the profession, or an experienced practitioner, we offer engineering and construction training courses that satisfy knowledge requirements across the breadth of infrastructure, as well as training on skills such as business and project management.</p>
+            <img src="./image/iso-class.webp" alt="iso bim class">
+        </section>
+        <hr>
+        <section class="courses">
+            <div class="courses-header">
+                <h3>Course Categories</h3>
+                <hr>
+                <p>Our training courses are available in multiple formats, including virtual classroom.</p>
+            </div>
+            <div class="course-items">
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Building Information Modelling</h4>
+                        <p class="course-description">Building Information Modelling (BIM) and working with digital twins is becoming an increasingly important part of project design, improving the efficiency of works and easing building maintenance requirements after construction is complete.</p>
+                    </div>
+                        <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/925f25fe-93ff-4d15-a92c-797cc88b93a7/BIM.jpg?width=936" alt="Building Information Modelling (BIM)"> 
+                </article>
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Accredited Project Managemen</h4>
+                        <p class="course-description">These online training courses provide construction professionals with the key project management skills needed for the delivery of successful projects. As well as the two longer courses preparing for exams, we offer a number of bite-size eLearning modules for quick information on specific topics.</p>
+                    </div>
+                    <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/aa26ec9e-790a-4628-b1f6-e0a385d9fec6/collaborative-working.jpg?width=936" alt="Accredited Project Management">
+                </article>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly make sure this section design sticks as much as possible to the one from the template. The images on Desktop version should be displayed in flex, their width, height, alignment,... should match the one of the template, spacing around and between the content elements should be close to the ones of the template, fonts, size, colors....
Make sure to check all details and improve your design

| Template Design | Your Design |
| -------------------- | ------------------ |
| ![Screenshot from 2023-01-05 20-08-59](https://user-images.githubusercontent.com/83042742/210852278-edfc0d76-4ca2-44ce-ae13-2aa41e6961a9.png) | ![Screenshot from 2023-01-05 20-15-48](https://user-images.githubusercontent.com/83042742/210852408-4b9143a7-8994-4589-8b42-0a19cbd33dd7.png) |